src/Controls/src/Core/Handlers/Shell/Windows/ViewToHandlerConverter.cs

```diff
@@ -53,7 +53,9 @@ public WrapperControl(View view)
 				_view = view;
 				_view.MeasureInvalidated += OnMeasureInvalidated;
 
-				FrameworkElement = view.ToPlatform(view.FindMauiContext()!);
+				// If we don't find the MauiContext and instead return the default, the view won't display
+				// Header and footers of a CollectionView don't have a parent when they're created (which is used to get the context), yet they still need to be displayed
+				FrameworkElement = view.ToPlatform(view.FindMauiContext(true)!);
```

Comment:
I feel like they should have a parent. Do they eventually have a parent? I think `CV.AddLogicalChild(Header)` should probably get called before we reach this point in the code, so I wonder if the header/footer aren't getting added as LC of the CV.

If this is a timing issue then we could put this code [here](https://github.com/dotnet/maui/pull/16870/files#diff-183f2ec157ab30b0ca65ae80a72e2910eeda7ce07cc3a7c5c3a5fdadd85bf1dfR63-R69)

Or possibly listen to "view.ParentSet" and once that Parent is set then set the FE